pkg/operator/controller/controller.go

```diff
@@ -71,19 +69,15 @@ const (
 
 var (
 	log = logf.Log.WithName("cdi-operator")
-
-	readyGauge = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Name: monitoring.MetricOptsList[monitoring.ReadyGauge].Name,
-			Help: monitoring.MetricOptsList[monitoring.ReadyGauge].Help,
-		})
 )
 
 func init() {
-	metrics.Registry = prometheus.NewRegistry()
-	metrics.Registry.MustRegister(readyGauge)
-	// 0 is our 'something bad is going on' value for alert to start firing, so can't default to that
```

Comment:
Yes here: https://github.com/kubevirt/containerized-data-importer/pull/3009/files#diff-e492805ba442ae8a4a1872f4650b03454bad344f0aa1b220f9015d1253a5beb3R80

Reply:
That initializes it with 0, not -1, no?

Reply:
correct, it's checked by me and joao and we decided to initialize to 0, no reason to use -1.

Reply:
I think this means the alert will start firing during CDI installation, which we were trying to avoid. Are we sure about that?

Reply:
Yes we checked it manually and the value as expected and no alerts during install.

Reply:
So if I read correctly the alert is going to fire until we reach https://github.com/kubevirt/containerized-data-importer/blob/ba75d9b557afeda68c471d356305e85bf633b646/pkg/operator/controller/controller.go#L223 which may take a while. What am I missing?

Reply:
I really don't know, you can check it manually and see, I don't think in the first place it was needed to be honest. the metrics crated after installation from what I understand.

adding @machadovilaca He might know to explain it better then me

Reply:
It might go very quick locally, so you don't catch it firing. on a real cluster, it may move slower with more errors and then you'll hit the alert during installation

Reply:
Joao explain that there is no reason to use -1 anymore, I didn't hit the alert and I install more then 5 times but if you strongly think it's a bug let wait for Joao to come back to work and see (: 

I can set it to -1 but I don't think it's needed.

Reply:
when thinking about it maybe you are right, let me fix it.

Reply:
I missed this one and tend to agree with Alex, as it may take > 5m until CDI becomes Available (Ready). We originally assumed the metric will be -1 until Available, as just when Progressing == false (some error) we set it to 0.

Reply:
Fixed and tested manually @akalenyu (: